pandas/core/window/rolling.py

```diff
@@ -1425,9 +1512,25 @@ def mean(self, *args, **kwargs):
 
     Parameters
     ----------
+    engine : str, default None
```

Comment:
can you do a shared doc?

Reply:
I can do this as a follow up. I have a separate branch I'm working on that is refactoring the docstrings to use the doc decorator.